src/f32/funcs.rs

```diff
@@ -1,3 +1,6 @@
+#[cfg(target_arch = "spirv")]
+use super::spirv::MathExt;
```

Comment:
I guess you might eventually put this in its own crate so other libraries can use it?

Reply:
yes we do have it in our `spirv-std` also: https://github.com/EmbarkStudios/rust-gpu/blob/a4a34243c0589aadf7496346a8c871d4c2f30c10/spirv-std/src/math_ext.rs

But it is not published yet but leaning towards simply publishing that and using it here